src/app/selectors.js

```diff
@@ -67,24 +67,14 @@ export const getLoginError = (state) => state?.root?.loginError;
   @param {Object} state The application state.
   @returns {Object|Null} The juju api instance or null if none found.
```

Comment:
Should this comment be updated to reflect that this now returns more than one instance?